roles/wisdom/wisdom_llm_load_test_multiplexed/templates/llm-load-test-config.json.j2

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
I personally prefer YAML files for configuration,
- yaml => human readable
- json => faster to parse
so I use yaml when it needs to be processed by humans, and json when it's a large file, as would really take time to parse it in Yaml

Reply:
the file can be transformed on the fly from yaml to json, if mandatory for the tool parsing it